validator/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for validator.js v5.7.0
+// Type definitions for validator.js v6.2.1
```

Comment:
This version number should only be `major.minor` - can you add a tslint.json with the following content?

```json
{ "extends": "../tslint.json" }
```